packages/web/components/error/error-boundary.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ErrorInfo, ReactNode } from "react";
+
+interface ErrorBoundaryProps {
+  children: ReactNode;
+  fallback: ReactNode;
+}
+
+interface ErrorBoundaryState {
+  hasError: boolean;
+}
+
+class ErrorBoundary extends React.Component<
```

Comment:
Is there a rationale for using a class component? IMO we should only us FunctionComponent.